build/inline_acceleration/firewall/scripts/start_p4_ovs_fw.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/bash
+#!/usr/bin/sleep
+#Copyright (C) 2022 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+#Firewall-Inline-Aceleration v0.5
+
+set -e
+
+OVS_DIR=/root/vvemula/p4-ovs-ipdk/p4-ovs-ipdk/  # Set the installation path of P4-OVS 
```

Comment:
These are all hard-coded directories and should be made a bit more configurable.

Reply:
Removed the script. Added the content in README file.